### ISSUE TITLE
Homepage UI should be focused on search rather than listing

### ISSUE THREAD

**Issue by 3pointross** [brashrebel/wordcrash-theme#17]
Users coming to the site are going to be focused on where they hope to stay, listing all available hosts across the globe gives them a lot of irrelevant information they have to filter out.

I'd recommend removing the list (except maybe recently added hosts to promote interest?) and putting all the emphasis on selecting a state and city.

**Comment by brashrebel** [comment]
I've always thought that we'd move in this direction, especially once we had enough hosts that a list of covered cities wouldn't be underwhelming. I could envision lots of cool ways to handle this like a map with pins on it or just some search/select fields.

However, there are challenges. If we are going to have someone _search_ then we'll ideally want some sort of proximity considered results. Currently we're only collecting the text of a host's city. A truly helpful search would allow people to search by city or zip code or something and get results sorted by proximity with a miles radius option for limiting results. We don't have a way to do this currently. They could search for the text but that isn't super helpful because it leaves out neighboring cities which should actually be returned as relevant results. Someone search for New York City should see a host in Brooklyn and someone searching for Ann Arbor should see a host in Dexter. We'd need some more functionality built in to make that work. Do you have any ideas?

Or maybe you are just saying the home page should just emphasize the select fields that exist now. When you say "search" I think text inputs where a user concocts their own query. But if we're just using the select fields of what host data exists then no problem. I like it.

**Comment by 3pointross** [comment]
I agree on all points, I think my initial description lacked detail.

For now I'm suggesting we keep the select fields with less emphasis on the current list of hosts. Eventually with enough hosts a radius based search would be neat but certainly wouldn't be helpful at this time.

I have a few other UI tweaks in mind and being a visual thinker/communicator, maybe it would be easiest if I did a quick mock-up for review and feedback?

**Comment by brashrebel** [comment]
@3pointross that would be amazing! While I totally trust your judgement and we'd probably just merge any PR you made, a mock-up here would also be cool just to open our discussion up.

**Comment by brashrebel** [comment]
And @3pointross you are a collaborator so you don't actually need any approval for committing to the repo. No PR needed unless you want it reviewed by others. Just commit away!

**Comment by 3pointross** [comment]
Thanks for the trust! I'll work on a mock-up for collaboration purposes. I'd rather get input from others on the team to make sure my thinking is inline with the vision of the project instead of suggesting changes in a vacuum. If my suggestions look good I'll get to work on them.
